### PR TITLE
CQLHS-473 | project evaluations controller post method now has data validation

### PR DESCRIPTION
**Pull request by ekm6**

### CODE REVIEW

File: Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Models/Request/EvaluationSubmissionRequest.cs [hunk 1]
```diff
@@ -1,25 +1,41 @@
...
-        public Guid Guid { get; set; }
+
+        [Required]
         public ProjectRole ReviewerProjectRole { get; set; }
+
+        [Required, Id]
         public int ReviewerId { get; set; }
+
+        [Required]
         public ICollection<EvaluationResponseRequest> QuestionResponses { get; set; }
```
Comment (tonysulfaro) [inline Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Models/Request/EvaluationSubmissionRequest.cs:24]: did the fields on EvaluationResponseRequest also get the required fields?

Comment (ekm6) [reply Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Models/Request/EvaluationSubmissionRequest.cs:24]: yerp

Comment (ekm6) [reply Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Models/Request/EvaluationSubmissionRequest.cs:24]: yerp

Comment (tonysulfaro) [reply Benchmark.WebApi/Benchmark.WebApi/Benchmark.WebApi/Models/Request/EvaluationSubmissionRequest.cs:24]: 👀

**Pull request merged**